py/client/pydeephaven/_arrow_flight_service.py

```diff
@@ -46,3 +46,14 @@ def do_get_table(self, table: Table) -> pa.Table:
             return reader.read_all()
         except Exception as e:
             raise DHError("failed to perform a flight DoGet on the table.") from e
+
+    def do_exchange(self):
+        try:
+            desc =  pa.flight.FlightDescriptor.for_command(b"dphn")
+            options = paflight.FlightCallOptions(headers=self.session.grpc_metadata)
+            writer, reader = self._flight_client.do_exchange(desc, options)
+            return writer, reader
+
+        except Exception as e:
+            raise DHError("failed to perform a flight DoExchange on the table.") from e
+            
```

Comment:
"on the table" seems to be not what the method does?

Reply:
Oh, I see what you mean. OK. I can try to change it